package.json

```diff
@@ -170,6 +172,7 @@
     "enquirer": "2.3.6",
     "enzyme": "3.10.0",
     "enzyme-adapter-react-16": "1.15.2",
+    "esbuild-loader": "2.18.0",
```

Comment:
sure

Reply:
@Hotell @ecraig12345  can we come to an agreement on this ? I'm getting directly opposite suggestions here in this PR

Reply:
> and won't even be used at all right now

it will be added soon probably, so person adding it will run into this scenario which I tried to point out so it is addressed. 

We can skip this for now to get this merged and I should probably go back to use https://conventionalcomments.org/ to clearly state the intention/importance of PR comments 😅

Reply:
@ecraig12345 I've removed the depenency on `esbuild`